Segment-Nielsen-DCR/Classes/SEGNielsenDCRIntegration.m

```diff
@@ -46,7 +46,8 @@
 {
     NSString *contentLengthKey = settings[@"contentLengthPropertyName"];
     NSString *contentLength;
-    if ([contentLengthKey length] > 0) {
+    NSString *customContentLength = src[contentLengthKey];
+    if ([contentLengthKey length] > 0 && customContentLength) {
```

Comment:
Generally LGTM! Question - what if the user passes an incorrect type as a value, such as an empty object, array, or INT? Not sure if we should be checking for those situations as well. Thanks!